api/v1alpha1/dnsrecord_types.go

```diff
@@ -36,23 +36,36 @@ const HttpsProtocol Protocol = "HTTPS"
 // the listeners assigned to the target gateway
 type HealthCheckSpec struct {
 	// Port to connect to the host on. Must be either 80, 443 or 1024-49151
+	// Defaults to port 80
```

Comment:
should we perhaps default to 443 (IE default to secure)?

Reply:
Sure. Done